### PR TITLE
Rest keep alive

### PR DESCRIPTION
**Pull request by sadysnaat**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/session/session.go [hunk 1]
```diff
@@ -213,8 +214,23 @@ func clearCache(sessionKey string) {
 }
 
 // newManager creates a Manager that encompasses the REST Client for the VSphere tagging API
-func newManager(ctx context.Context, client *vim25.Client, user *url.Userinfo) (*tags.Manager, error) {
+func newManager(ctx context.Context, logger logr.Logger, sessionKey string, client *vim25.Client, user *url.Userinfo, feature Feature) (*tags.Manager, error) {
```
Comment (jzhoucliqr) [inline pkg/session/session.go:217]: Thanks Deepak. Shall we open PR to upstream and have it reviewed there before merge?

Comment (jzhoucliqr) [reply pkg/session/session.go:217]: does this functionality exist in our v1a3 branch?

Comment (sadysnaat) [reply pkg/session/session.go:217]: we don't need it in v1alpha3 as this tagmanager is initialized for vspherefailuredomain

Comment (sadysnaat) [reply pkg/session/session.go:217]: > Thanks Deepak. Shall we open PR to upstream and have it reviewed there before merge?

Ack

**Pull request merged**